flarefly/utils.py

```diff
@@ -44,3 +44,70 @@ def __init__(self, text, level):
             print(f'\n\n{text}\n\n')
         else:
             print(text)
+
+pdg_dict = {'22': {'name': 'photon', 'mass': 0.0},
```

Comment:
the key does not need to be a string (but we can live with that for the moment`)